esp/espcommon/espcommon.cpp

```diff
@@ -52,8 +52,12 @@ static std::vector<__uint64> getExecutionProfileBuckets(const char *profileOptio
     return buckets;
 }
 
-// Registers a profiling metric for an ESP service method
-ESPCOMMON_API std::shared_ptr<hpccMetrics::ScaledHistogramMetric> registerProfilingMetric(const char *histogramMetricName, const char *desc, const char *profilingOptions)
+// Generate metric name and register it for the indicated service method
```

Comment:
Note: You will probably require the original function later on.  This should probably be an additional function which calls the old one.